.github/CODEOWNERS

```diff
@@ -0,0 +1,47 @@
+# Unowned specs
+/ @mongodb/dbx-spec-reviewers-unowned
+
+# APM
+/source/command-monitoring/ @mongodb/dbx-spec-owners-apm
+
+# Authentication
+/source/auth/ @mongodb/dbx-spec-owners-auth
+
+# Change Streams
+/source/change-streams/ @mongodb/dbx-spec-owners-change-streams
+
+# Client Side Encryption
+/source/client-side-encryption/ @mongodb/dbx-spec-owners-fle
+
+# CMAP
+/source/connection-monitoring-and-pooling/ @mongodb/dbx-spec-owners-cmap
+
+# CRUD
+/source/crud/ @mongodb/dbx-spec-owners-crud
+/source/driver-bulk-update.rst @mongodb/dbx-spec-owners-crud
+
+# Retryability
+/source/retryable-reads/ @mongodb/dbx-spec-owners-retryability
+/source/retryable-writes/ @mongodb/dbx-spec-owners-retryability
+
+# SDAM
+/source/server-discovery-and-monitoring/ @mongodb/dbx-spec-owners-sdam
+
+# Server Selection
+/source/server-selection/ @mongodb/dbx-spec-owners-server-selection
+
+# Sessions
+/source/sessions/ @mongodb/dbx-spec-owners-sessions
+
+# Transactions
+/source/transactions/ @mongodb/dbx-spec-owners-transactions
+/source/transactions-convenient-api/ @mongodb/dbx-spec-owners-transactions
+
+# Unified Test Format
+/source/unified-test-format/ @mongodb/dbx-spec-owners-unified-test-format
+
+# URI Options
+/source/uri-options/ @mongodb/dbx-spec-owners-uri-options
+
+# Workflows and code ownership file
+/.github/ @mongodb/dbx-leadership
```

Comment:
> Furthermore, teams allow to configure review assignments to request a review from a team member directly instead of putting the request on the team.

That's good, everyone in dbx-leadership seems like too much for reviewing changes in the workflows.

Reply:
Even then, only a single lead needs to approve the PR on behalf of the team, but it creates noise for all leads.